job/storage/redis/redis.go

```diff
@@ -19,8 +19,14 @@ type DB struct {
 }
 
 // New instantiates a new DB.
-func New(address string, password redis.DialOption) *DB {
-	conn, err := redis.Dial("tcp", address, password)
+func New(address string, password redis.DialOption, passinterface int) *DB {
```

Comment:
@ajvb Changing the signature of `New` by requiring an additional parameter is (obviously) a breaking change. Is that OK?